Gemfile

```diff
@@ -54,3 +54,7 @@ gem "devise", "~> 4.8"
 gem "react-rails", "~> 2.6"
 
 gem "bootstrap", "~> 5.1"
+
+gem 'rack-cors', :require => 'rack/cors'
+
```

Comment:
You don't need `cors` for this project as you are not making a request across origin.